test/common/testlib.py

```diff
@@ -631,8 +631,7 @@ def login_and_go(self, path=None, user=None, host=None, authorized=True):
 
         # HACK https://bugzilla.redhat.com/show_bug.cgi?id=1461893
         # selinux errors while logging in via ssh
-        'type=1401 audit(.*): op=security_compute_av reason=bounds scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=unconfined_u:system_r:svirt_lxc_net_t:s0-s0:c0.c1023 tclass=process perms=transition,sigchld,sigstop,signull,signal,getattr',
-        'type=1401 audit(.*): op=security_compute_av reason=bounds scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=unconfined_u:unconfined_r:svirt_lxc_net_t:s0-s0:c0.c1023 tclass=process perms=transition,sigchld,sigstop,signull,signal,getattr',
+        'type=1401 audit(.*): op=security_compute_av reason=bounds .* tclass=process perms=transition.*',
```

Comment:
I feel like this is generalized a bit too much. Also, since this is for a different distro, I think we should file a separate bug.